esmvaltool/diag_scripts/autoassess/land_surface_surfrad/surfrad.py

```diff
@@ -7,19 +7,44 @@
 import iris
 
 from esmvalcore.preprocessor import regrid
+from esmvaltool.diag_scripts.shared._base import ProvenanceLogger
 from esmvaltool.diag_scripts.shared._supermeans import get_supermean
 
 
-def land_surf_rad(run):
-    """
-    Compute median absolute errors against CERES-EBAF data.
+def get_provenance_record(plot_file, caption, run):
+    """Create a provenance record describing the diagnostic data and plot."""
+    record = {
+        'caption': caption,
+        'statistics': ['mean'],
+        'domains': ['global'],
+        'plot_type': 'map',
```

Comment:
surfrad diagnostic only produces the metrics plot, so 'map' isn't right. Not sure how best to describe the metrics plot, perhaps 'point', or 'bar'? Is there a list of allowed values for 'plot_type'?

Reply:
sames here, `autoassess metric` is nicely self-explanatory

Reply:
As above.